includes/rc.local

```diff
@@ -25,7 +25,7 @@ if [ ! -f $FLAG ]; then
   systemctl enable openhab2.service
   systemctl start openhab2.service
 
-  if [ -f "/home/pi/openHABian-install-failed" ]; then
+  if [ -f "/home/`ls /home`/openHABian-install-failed" ]; then
```

Comment:
Not for now. I will look into that with a later PR. We should really stay with "Password change function" in this one. ;)